reactive-schema.sql

```diff
@@ -0,0 +1,24 @@
+USE eventuate;
```

Comment:
What is reactive about this schema?

Reply:
I meant schema for reactive tests. I will update.